README.md

```diff
@@ -158,7 +163,40 @@ yarn start
 open localhost:3000
 ```
 
+## Docker Quickstart
+To start Unity Catalog in Docker Compose in one command, install the latest version of [Docker Desktop](https://www.docker.com/products/docker-desktop/) and run the following:
+
+```sh
+docker compose up -d
+```
+
+This starts the Unity Catalog UI and backend. You can access the UI at http://localhost:3000 and the backend at http://localhost:8080.
+
+To use the Unity Catalog CLI, attach to a shell in the backend container:
+
+```sh
+docker exec -it unitycatalog-backend-1 /bin/bash
```

Comment:
Oh, no, I'm sure that `unitycatalog-backend-1` will soon be the number 1 for many to replace with something more meaningful. Can you replace `backend-1` and likely make `unitycatalog` shorter (say `uc`)? 🙏

Reply:
This is default from Docker Compose, which creates containers according to the following: <docker-compose-name>_<service-name>_<service_container_cardinatlity>.

I think it's better to give the complete name: `unitycatalog` and be more clear about the container throughout its lifecycle, than to use shorthands.

Reply:
Let's rename `backend` to be `server` to give more clarity.